hack/generator/azure-cloud.yaml

```diff
@@ -6,3 +6,6 @@ typefilters:
   - action: exclude
     version: '*preview'
     because: preview SDK versions are excluded by default
+  - action: exclude
+    group: 'deploymenttemplate'
+    because: This is the "container" group that holds references to all of the other groups and as such doesn't make sense to generate
```

Comment:
Given the way the message is output in logs, suggest you start it with lowercase: `this is the ...` #nit